src/mmw/mmw/settings/layer_settings.py

```diff
@@ -143,7 +143,7 @@
     },
     {
         'code': 'nhd_streams_v2',
-        'display': 'National Stream Network',
+        'display': 'Continental US Medium Resolution Stream Network',
```

Comment:
Consider adding a line break here as having it all on a single line seems to expand the width of the layer selector box. We do something similar with the DRB names.